index.html

```diff
@@ -39,7 +43,7 @@ <h1 class="title">I'm Mariane.<br/> Glad to see you!</h1>
                     <div class="header-handle">
                         <div class="connect">LET'S CONNECT</div>
                         <div class="social-media">
-                            <img class="media-items" src="Microverse-Student-Project-1/social media icons.png" alt="social-media">
+                            <img class="media-items" src="Microverse-Student-Project-1/social media icons.png">
```

Comment:
- [ ] I noticed you removed all `alt` attributes for the images as you were implementing the changes suggested by the previous reviewer. Kindly return them. Using `alt` attributes when using the `img` tags makes it easy for screen readers to capture images for people who use screen readers to navigate web applications. 👍